Common/Sources/View/Error/ErrorView.swift

```diff
@@ -64,6 +64,17 @@ public final class ErrorView {
     ///
     /// - Parameter handler: to handle business error
     private func businessError(_ handler: ((UIAlertAction) -> Void)? = nil) {
+        if let errorMessage = error.getHyperwalletErrors()?.errorList?.first?.message {
```

Comment:
use if let error = error.getHyperwalletErrors()?.errorList?.first and then use error.message, error.fieldName and error.code. That's more cleaner